hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/MultiDeleteRequestUnmarshaller.java

```diff
@@ -68,6 +56,10 @@ public MultiDeleteRequest readFrom(Class<MultiDeleteRequest> type,
       Type genericType, Annotation[] annotations, MediaType mediaType,
       MultivaluedMap<String, String> httpHeaders, InputStream entityStream) {
     try {
+      JAXBContext context = JAXBContext.newInstance(MultiDeleteRequest.class);
+      SAXParserFactory saxParserFactory = SAXParserFactory.newInstance();
+      saxParserFactory.setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true);
```

Comment:
We might not need to instantiate this for every request. Could you help check whether instantiating only `XMLReader` for each request is enough?

Reply:
@ivandika3 Thanks for the review.
Yes, this modification solves the problem as well, I've done a test, please see the new code.